install-dev/models/install.php

```diff
@@ -114,21 +116,14 @@ public function generateSettingsFile(
                     'database_engine' =>  $database_engine,
                     'cookie_key' => $cookie_key,
                     'cookie_iv' =>  $cookie_iv,
+                    'new_cookie_key' => $key->saveToAsciiSafeString(),
```

Comment:
I agree with creating a new key for this but I'm trying to come up with a better name. I don't like having *new* in names because it's not really new forever. Maybe something like `cookie_salt`? I dunno <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://immhpnclomdloikkpcefncmfgjbkojmh/emoji-data/sheet_apple_64.png);background-position:65% 45%;background-size:4100%" title="smile"></span></span> 
Maybe even cookie_key_2 :/